docs/AnnotationsApi.md

```diff
@@ -0,0 +1,164 @@
+# \AnnotationsApi
```

Comment:
Weird, does this backslash actually have meaning - do only API file docs get this, rather than types?

Reply:
Yeah, it looks like all the `*API.md` files have this in them like this, with the `\`. I have no idea what ie means, but we're also going away from this with v8, so I'm not too concerned. It matches what was happening in other files